### PR TITLE
Match IC

### PR DESCRIPTION
**Pull request by bpalmeiro**
Minimal update to deal with IC updates

### CODE REVIEW

File: krcal/core/correction_functions.py [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
 from invisible_cities.reco.corrections import correct_geometry_
-from invisible_cities.reco.corrections import norm_strategy
+from invisible_cities.types.symbols    import NormStrategy
 from invisible_cities.reco.corrections import get_normalization_factor
 
 
-def e0_xy_correction(map        : ASectorMap                         ,
-                     norm_strat : norm_strategy   = norm_strategy.max):
+def e0_xy_correction(map         : ASectorMap                         ,
+                     norm_strategy: NormStrategy   = NormStrategy.max):
```
Comment (gonzaponte) [inline krcal/core/correction_functions.py]:
```suggestion
def e0_xy_correction(map          : ASectorMap                         ,
                     norm_strategy: NormStrategy = NormStrategy.max):
```

File: krcal/core/kr_parevol_functions.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def computing_kr_parameters(data       : pd.DataFrame,
 
     ## lt and e0
     geo_correction_factor = e0_xy_correction(map =  emaps                         ,
-                                             norm_strat = norm_strategy.max)
+                                             norm_strategy = NormStrategy.max)
```
Comment (gonzaponte) [inline krcal/core/kr_parevol_functions.py]: align =

File: krcal/map_builder/map_builder_functions.py [hunk 3]
```diff
@@ -326,7 +326,7 @@ def band_selector_and_check(dst       : pd.DataFrame,
         input_mask = [True] * len(dst)
     else: pass;
 
-    emaps = e0_xy_correction(boot_map, norm_strat  = norm_strat)
+    emaps = e0_xy_correction(boot_map, norm_strategy  = norm_strat)
```
Comment (gonzaponte) [inline krcal/map_builder/map_builder_functions.py]:
```suggestion
    emaps = e0_xy_correction(boot_map, norm_strategy = norm_strat)
```

**Review by gonzaponte — COMMENTED** [review]
Cosmetics only. Does this PR assume IC v2?

**Comment by bpalmeiro** [comment]
It assumes the IC `master`. I guess I should try with `v2-development`, right?

**Comment by gonzaponte** [comment]
Yes, but it will become master in the coming days

**Comment by bpalmeiro** [comment]
As a crosscheck, I ran with that branch, and no errors in the test suit or map production

**Review by gonzaponte — APPROVED** [review]
Code updated to work with IC 2.0. Good job!

**Pull request merged**